Path-Check/covid-safe-paths
# SAF-210 - Fix debug build error with apache commons-io lib

…ignoring apache commons lib



#### Description:

On the latest develop build (as of May 6, 2020), installing a debug (Android) APK would result in the app throwing a "red box error" (see attached image), that indicated we were missing a crucial dependency. In a release build, this would manifest as the app crashing.

This fix patches the @mauron85/background-geolocation-android module, per the pull request here: https://github.com/mauron85/background-geolocation-android/pull/31, using patch-package, to address its conflicting apache commons dependency, rather than excluding that dependency for the whole project.


#### Linked issues:


#### Screenshots:
<img width="317" alt="Screen Shot 2020-05-06 at 3 44 17 PM" src="https://user-images.githubusercontent.com/2692908/81309678-35590a80-9051-11ea-9f7d-43b0635ca0dc.png">

#### How to test:

Install an APK and ensure that the app does not crash on loading, or display a red box error (see attached image)
